Documentation update. (#18)
npatsakula/bitmap: README.md

# Dynamic-sized lightweight bitmap

## Explanation

This zero-dependencies solution has been written with the aim to fill

> the a niche of a simple(zero-dependencies) and hi-performance solutions

Reviewer: ```suggestion
the a niche of a simple and high-performance solutions
```

> ## Explanation
> 
> This zero-dependencies solution has been written with the aim to fill
> the a niche of a simple(zero-dependencies) and hi-performance solutions
> with a nice API.
> 

Reviewer: Я бы выкосил весь блок Explanation и заменил бы на 
```suggestion
Simple, fast and easy to use bitmap.
```

## Example

```rust
use dyn_bitmap::DynBitmap;
use std::io::Write;

let source = [true, false, false].iter()
    .cycle()
    .take(256);

// You can construct bitmap from iterator with `Type = bool` or
// construct bitmap manually with `contained/1`.
//

> // `DynBitmap` has hi-performance `from_iter` implementation,

Reviewer: ```suggestion
// `DynBitmap` has high-performance `from_iter` method,
```

> // it's preferable to `contained/1`.

Reviewer: ```suggestion
// which is preferable to `contained/1`.
```

let mut bitmap: DynBitmap = source.copied().collect();

// You can set value of bitmap using `set/2` function.
// It returns additional information in case of an error.
bitmap.set(true, 1).unwrap();
// You can check value of some particular bit using `get/1`.

> // Note, that bit's index starts from `0`.

Reviewer: ```suggestion
// Note, that bit index starts from `0`.
```
